src/coreclr/jit/objectalloc.cpp

```diff
@@ -1139,6 +1139,25 @@ bool ObjectAllocator::CanLclVarEscapeViaParentStack(ArrayStack<GenTree*>* parent
                     canLclVarEscapeViaParentStack =
                         !Compiler::s_helperCallProperties.IsNoEscape(comp->eeGetHelperNum(asCall->gtCallMethHnd));
                 }
+                else if (asCall->IsSpecialIntrinsic())
+                {
+                    // Some known special intrinsics don't escape. At this moment, only the ones accepting byrefs
```

Comment:
byrefs for non-GC types....

Reply:
I'll add that to the comment as part of a different PR to avoid re-running CI 🙂